Gemfile

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+
+gem 'asciidoctor', '~> 2.0', '>= 2.0.20'
+
+# Uncomment for ability to render pdf:
+# gem 'asciidoctor-pdf', '~> 2.0', '>= 2.0.20'
```

Comment:
Any reason we are leaving the pdf commented and removing all the pdf but leaving 'Markdown to AsciiDoc' below? Comparing with https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/Gemfile

Reply:
It is commented in https://github.com/openstack-k8s-operators/openstack-operator/blob/main/Gemfile#L6 so I left it as it is.

Reply:
Dropped the commented piece from Gemfile as asciidoctor-pdf is not used in the existing make target.